pkgs/tools/networking/libreswan/default.nix

```diff
@@ -6,7 +6,7 @@
 
 let
   optional = stdenv.lib.optional;
-  version = "3.31";
+  version = "3.32";
   name = "libreswan-${version}";
```

Comment:
Nitpick: it seems there is no reason why `optional`, `version` and `name` is defined here.

I suggest removing these 3 lines. Then adding `pname` and `version` to `mkDerivation` below and expanding `optional` into `stdenv.lib.optional` in `mkDerivation`.

This is just a matter of taste, though. That's why I am just adding as an extra comment, not part of the review.